go/client/cmd_pgp_gen.go

```diff
@@ -68,6 +73,32 @@ func (v *CmdPGPGen) Run() (err error) {
 	return err
 }
 
+// Duplicates the test in libkb.PGPGenArg.CreatePGPIDs() for interactive input
+var CheckPGPID = libkb.Checker{
+	F: func(s string) bool {
+		if !strings.Contains(s, "<") && libkb.CheckEmail.F(s) {
+			return true
+		}
+		_, err := libkb.ParseIdentity(s)
+		return err == nil
+	},
+	Hint: "2-12 letter id or email, or pgp style id",
+}
+
+func (v *CmdPGPGen) propmptPGPIDs() (err error) {
+	prompt := "Enter default ID"
```

Comment:
yeah, when people do this in interactive mode, we're putting them on rails, so to speak, not giving them all the possibilities of PGP. they should be entering a full name once, a regular email address, etc.